caluma/form/jexl.py

```diff
@@ -17,11 +19,19 @@ def visit_Transform(self, transform):
 
 
 class QuestionJexl(JEXL):
-    def __init__(self, answer_by_question=None, form=None, **kwargs):
-        answer_by_question = answer_by_question if answer_by_question else {}
+    def __init__(self, validation_context=None, **kwargs):
+        answer_by_question = validation_context["answers"] if validation_context else {}
+
         super().__init__(**kwargs)
 
-        self.context = Context({"form": form})
+        context_data = None
+        if validation_context:
+            context_data = {**validation_context}
```

Comment:
Isn't `validation_context` a dict already?

Reply:
Yes. But I'm modifying it a few lines further down, which I don't want to propagate to the caller. Thus, a copy is created

Reply:
:+1: